openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -78,64 +79,59 @@ def _get_response(query_param, endpoint=ENDPOINT, retries=RETRIES):
         return response_json, total_images
 
 
+def get_int_value(data, key):
+    value = data.get(key)
+    if value is not None and value != "":
+        return int(value)
+    return None
+
+
+def _handle_batch_item(data):
+    license_ = data.get("share_license_status", "").lower()
+    if license_ != "cc0":
+        return None
+    foreign_id = data.get("id")
+    if foreign_id is None:
+        return None
+    image = _get_image_type(data.get("images", {}))
+    if image is None or image.get("url") is None:
+        return None
+
+    if data.get("creators"):
+        creator_name = data.get("creators")[0].get("description", "")
+    else:
+        creator_name = ""
+
+    return {
+        "foreign_identifier": f"{foreign_id}",
```

Comment:
Why is the string interpolation necessary here? Could we just do `str(foreign_id)` if it needs to be converted to a string?

Reply:
WOAH, that's so good to know!! Thanks for sharing 😄